dev/src/components/navbar/NavBarCourse.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { useHistory } from "react-router";
+import { Enrollment } from "../../../proto/ag/ag_pb";
+import { Status } from "../../consts";
+import { useActions, useAppState } from "../../overmind";
+import NavBarLabs from "./NavBarLabs";
+import NavBarTeacher from "./NavBarTeacher";
+
+const NavBarCourse = ({ enrollment }: { enrollment: Enrollment }): JSX.Element => {
+    const state = useAppState()
+    const actions = useActions()
+    const history = useHistory()
+    // If selected, used to show teacher / student navbar for that course
```

Comment:
I'm a bit confused by the comment and the `selected` variable. Can you clarify this comment. The code sets `selected` to true if the courseID of the passed-in enrollment equals the active course.